schemas/general/contentLicense0.json

```diff
@@ -0,0 +1,12 @@
+{
+  "classId": "Content License",
+  "newProperties": [
+    {
+      "name": "Value",
+      "description": "The license of which the content is originally published under.",
```

Comment:
Again, I think better to remove "originally" here.

Reply:
Same:
Good point, but this is already live...